web/auth.html

```diff
@@ -18,7 +18,7 @@
 
   function getOnAuthRedirectUrl() {
     let redirectUrl = window.location.href.replace('/auth.html', '/#/onAuthRedirect');
-    return redirectUrl.replace('/web', '');
+    return redirectUrl;
```

Comment:
we should separate this fix in other PR and test carefully with some platform (saas and normal)

Reply:
Move to this ticket #2067